content/en/apps/reference/app-settings/_index.md

```diff
@@ -29,20 +35,21 @@ The following settings do not need to be specified. They should only be defined
+| app_url                          | The URL of the app, eg: "https://demo.app.medicmobile.org"                                                                                                                                                                                                                                                                                                                                                                                                                            |         | 3.10.0  |
+| task_days_overdue                | Display number of overdue days in tasks list                                                                                                                                                                                                                                                                                                                                                                                                                                          | false   | 3.13.0  |
+| languages                        | Array of objects with `locale` and `enabled` properties representing respectively the 2 or 3 letter language code and whether that language should be enabled.  <br/>If unset it falls back to the previous behavior of relying on the `enabled` property of each translation document `messages-XX.properties`. This fallback behavior is now deprecated and will be removed in the next major version (5.0). This `languages` configuration property will be required for CHT 5.0+. |         | 4.2.0   |
+| place_hierarchy_types            | Array of contact types' IDs, should match the ones defined in `contact_types`. This is used to define the Place Filter's options in Reports tab.                                                                                                                                                                                                                                                                                                                                      |         | 2.15.0  |
+| assetlinks                       | Array of [Digital Asset Links](https://developers.google.com/digital-asset-links/v1/getting-started) definitions. This is used to associate your CHT instance's domain to your Android app to verify app links.                                                                                                                                                                                                                                                                       |         | 4.7.0   |
```

Comment:
Looking at this more, I am not sure we want to include `assetlinks` in this table.  None of the other app_settings properties that have their own docs pages (e.g. `forms`, `permissions`, etc) seem to be included here.  I don't see any particular problem with listing `assetlinks` here, though, so I am fine with whatever you decide is best.  

(Though, if you do leave this in, I would recommend that we switch to link to point to the `app-settings/assetlinks` page instead of the Google docs. Just want it to be clear that we have more docs about this property...)

Reply:
I'd rather leave it in as this is the section for optional properties and `assetlinks` is very much optional. I did update the link though, good thinking!